src/Components/Web/src/EventHandlers.cs

```diff
@@ -43,19 +43,19 @@ namespace Microsoft.AspNetCore.Components
     [EventHandler("onkeypress", typeof(UIKeyboardEventArgs))]
 
     // Input events
-    [EventHandler("onchange", typeof(UIChangeEventArgs))]
-    [EventHandler("oninput", typeof(UIChangeEventArgs))]
-    [EventHandler("oninvalid", typeof(UIEventArgs))]
-    [EventHandler("onreset", typeof(UIEventArgs))]
-    [EventHandler("onselect", typeof(UIEventArgs))]
-    [EventHandler("onselectstart", typeof(UIEventArgs))]
-    [EventHandler("onselectionchange", typeof(UIEventArgs))]
-    [EventHandler("onsubmit", typeof(UIEventArgs))]
+    [EventHandler("onchange", typeof(ChangeEventArgs))]
+    [EventHandler("oninput", typeof(ChangeEventArgs))]
+    [EventHandler("oninvalid", typeof(EventArgs))]
+    [EventHandler("onreset", typeof(EventArgs))]
+    [EventHandler("onselect", typeof(EventArgs))]
+    [EventHandler("onselectstart", typeof(EventArgs))]
+    [EventHandler("onselectionchange", typeof(EventArgs))]
+    [EventHandler("onsubmit", typeof(EventArgs))]
 
     // Clipboard events
-    [EventHandler("onbeforecopy", typeof(UIEventArgs))]
-    [EventHandler("onbeforecut", typeof(UIEventArgs))]
-    [EventHandler("onbeforepaste", typeof(UIEventArgs))]
+    [EventHandler("onbeforecopy", typeof(EventArgs))]
+    [EventHandler("onbeforecut", typeof(EventArgs))]
+    [EventHandler("onbeforepaste", typeof(EventArgs))]
     [EventHandler("oncopy", typeof(UIClipboardEventArgs))]
```

Comment:
Do you still think the `UI` prefix makes sense on the other event names?

Since this is now in `.Web`, I would think names like `DOMClipboardEventArgs` would be a more direct expression of what this is.

Or if we don't want the `DOM` prefix (e.g., because we are shy about the implication that it contains *exactly* the same info) we could just not have a prefix now. `ClipboardEventArgs` would also be fine.

Reply:
I like the idea of this. Do you think this needs more discussion or would we just do it.

Also according to the actual guidelines it would `DomClipboardEventArgs` which is a little bit ugly. I'm going to proceed with this PR as is and we can discuss offline.